### ISSUE TITLE
Box problem

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#74]
```
What steps will reproduce the problem?
1. taking a box, which contains the thorwballs, with the plane
makes you get another plane instead of it


```

-----
Original issue reported on code.google.com by `alessand...@gmail.com` on 29 Sep 2010 at 12:47

**Comment by GoogleCodeExporter** [comment]
```
I thought this was a dupe, but couldn't find the older one to dupe against.

Anyway, known issue, already fixed in 0.9.14

I encourage bug reporters to try 0.9.14 before reporting, but thanks for taking 
the time anyway.

```

-----
Original comment by `kyberneticist@gmail.com` on 29 Sep 2010 at 2:29
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified (crate gives item when holding plane)
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 8:06
* Changed state: **Verified**
